pandas/io/orc.py

```diff
@@ -52,3 +56,50 @@ def read_orc(
     with get_handle(path, "rb", is_text=False) as handles:
         orc_file = orc.ORCFile(handles.handle)
         return orc_file.read(columns=columns, **kwargs).to_pandas()
+
+
+def to_orc(
+    df: DataFrame,
+    path: FilePath | WriteBuffer[bytes] | None = None,
```

Comment:
How is `None` handled?

Reply:
Good catch! It has been fixed.

Reply:
You are welcome to add overloads here as well but would be less important.